src/homework/lesson_6/ChatServer.java

```diff
@@ -0,0 +1,35 @@
+package homework.lesson_6;
+
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class ChatServer {
+    private final String WAITING_CONNECT_CLIENT = "Ожидаем подключение клиента";
+    private final String CONNECTED_CLIENT = "Клиент подключился";
+
+    private final IChatViewModel viewModel;
+    private WriterSocket writer;
+
+    public ChatServer(IChatViewModel viewModel) throws Exception {
+        if (viewModel == null)
+            throw new Exception("ChatServer.viewModel is null");
+        this.viewModel = viewModel;
+    }
+
+    public void Start() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.